main.go

```diff
@@ -129,15 +132,21 @@ func main() {
 	if err != nil {
 		if exitError := new(exec.ExitError); errors.As(err, &exitError) {
 			exitCode := exitError.ExitCode()
+
+			if status, ok := exitError.Sys().(syscall.WaitStatus); ok && status.Signaled() {
+				logSignalAndExit(testRunner.Name(), status.Signal())
+			}
+
```

Comment:
`exitError.Sys()` will return something castable to `syscall.WaitStatus` - ie, the value of `ok` will be `true` - only on unixes, which i suspect will mean that on windows (the only notable non-unix that we might be expected to ship the test splitter to), behaviour will be undefined.

i think that this is probably totally fine, but i figured i'd point it out anyway. it'll just mean that when something is terminated with a signal on windows (does that even happen? i don't know) we'll likely return an exit code of -1, which again, i think is probably fine.

Reply:
Legend, thank you so much!